next.config.js

```diff
@@ -3,13 +3,23 @@ const withCSS = require("@zeit/next-css");
 const withTM = require("next-transpile-modules");
 
 const withMdx = require("@next/mdx")({
-  extension: /\.mdx?$/,
+  extension: /\.mdx?$/
 });
 
 const config = {
   target: "serverless",
   pageExtensions: ["jsx", "js", "bs.js", "mdx"],
-  transpileModules: ["bs-platform"].concat(bsconfig["bs-dependencies"])
+  transpileModules: ["bs-platform"].concat(bsconfig["bs-dependencies"]),
+  webpack: (config, { isServer }) => {
+    // Fixes npm packages that depend on `fs` module
```

Comment:
Any more details on that? Is this introduced by `refmt` JS?

Reply:
As written in `reason` package README:
> NOTE: refmt.js requires the node module fs, which of course isn't available on the web. If using webpack, to avoid the missing module error, put node: { fs: 'empty' } into webpack.config.js. See https://webpack.js.org/configuration/node/#other-node-core-libraries for more information.

Reply:
right, maybe we should add this reference as a comment with a link to the reason README 👍